.templates/GETTING_STARTED.md

```diff
@@ -0,0 +1,12 @@
+## Getting Started
+
+To get started with this repository, follow these steps:
+
+Clone this repository to your local machine.
+```sh
+git clone :git_project_path
+```
+Build and start containers. It may takes some time.
+```sh
+docker compose up -d
+```
```

Comment:
please addblank lines before and after code blocks https://github.com/DavidAnson/markdownlint/blob/main/doc/md031.md

Reply:
Done